writing-apps/our-first-app/the-build-system.md

```diff
@@ -69,6 +69,25 @@ If all went well, you should now be able to open your app from the Applications
 If you were about to add the "build" folder to your git repository and push it, stop! This binary was built for your computer and we don't want to redistribute it. In fact, we built your app in a separate folder like this so that we can easily delete or ignore the "build" folder and it won't mess up our app's source code.
 {% endhint %}
 
+## Uninstalling the application
+
+To uninstall your application, change into the build directory and we will use `ninja` once again to uninstall:
+
+```bash
+cd build
+sudo ninja uninstall
+```
+
+If all went well, you should see command output that shows files related to your application were removed.  Again, more details can be found in [Meson's documentation](https://mesonbuild.com/Manual.html).
+
+```bash
+...
```

Comment:
The manual removal commands still appear in the copy, and the code block is without any context or explanation. I think that if you wanna keep the manual instructions for any reason at least provide context about what those commands do.